packages/react-instantsearch/examples/next-app/package.json

```diff
@@ -2,12 +2,12 @@
   "name": "create-next-example-app",
   "scripts": {
     "dev": "next",
-    "build": "next build",
+    "build": "NODE_ENV=development next build",
```

Comment:
Hm indeed that babelrc is only setupping things for development. We should setup things as a whole and then provide some differences for production